README.md

```diff
@@ -138,21 +138,29 @@ Returns a summary of quota usage in Uppmax
 * couchdb
 * pprint
 
+
+### Samplesheet_converter_v1.0.py
+For the purpose of converting Illumina samplesheet that contains Chromium 10X indexes for demultiplexing. Headers and lines with ordinary indexes will be passed without any change. Lines with Chromium 10X indexes will be expanded into 4 lines, with 1 index in each line, and suffix 'Sx' will be added at the end of sample names.
+For details refer to https://github.com/chuan-wang/Samplesheet_converter
+#### Usage
+`python main.py -i <inputfile> -o <outputfile> -x <indexlibrary>`
```

Comment:
I think it will beneficial if you copy in this repo the index library file that you have in your repo sample_sheet_converter. 
You have wrote a great description of the script in your repo. My suggestion is that you do not duplicate the script in two different repositories, but you have it only in stand_alone_scripts. You can put you repo as a folder in the stand_alone so that you can have a complete README and the nice examples you have